documentation/src/main/asciidoc/chapter-2-set-up.asciidoc

```diff
@@ -254,6 +254,18 @@ Supported values are:
 If a component model is given for a specific mapper via `@Mapper#componentModel()`, the value from the annotation takes precedence.
 |`default`
 
+|`mapstruct.defaultInjectionStrategy`
+| The type of the injection in mapper via parameter `uses`.
```

Comment:
I think that we should mention somewhere that this is only is only used on annotated based component models such as CDI, Spring and JSR 330. Similar as we don in the Javadoc of `Mapper#injectionStrategy`

Reply:
Done